draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/FieldMessage.tsx

```diff
@@ -1,26 +1,26 @@
-import classnames from "classnames"
 import * as React from "react"
-import exclamationIcon from "@kaizen/component-library/icons/exclamation.icon.svg"
-
 import { Icon, Paragraph } from "@kaizen/component-library"
+import exclamationWhiteIcon from "@kaizen/component-library/icons/exclamation-white.icon.svg"
+import classnames from "classnames"
 import styles from "./styles.scss"
 
-export type FieldMessageStatus = "default" | "success" | "error"
+export type FieldMessageStatus = "default" | "success" | "error" | "caution"
+
 export type FieldMessageProps = {
   id?: string
   automationId?: string
-  message?: React.ReactNode
+  message?: string | React.ReactNode
   status?: FieldMessageStatus
   reversed?: boolean
   position?: "top" | "bottom"
 }
 
-type FieldMessage = React.SFC<FieldMessageProps>
+type FieldMessage = React.FunctionComponent<FieldMessageProps>
```

Comment:
You can move the value into L31 :)

Reply:
nice spot, always happy for less lines of code :)